app/templates/base_form.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+
+{% block head %}
+<title>{{ header }} | Stratus</title>
+
+<link rel="stylesheet" href="{{ static('vendor/chosen/chosen.min.css') }}">
+<link rel="stylesheet" href="{{ static('css/explore.css') }}">
+{% endblock %}
+
+{% block body %}
+<div class="content">
+  <div class="frame">
+    <div class="content">
+      <h2 class="text-primary">{{ header }}</h2>
+      {% if form.errors %}
+        <p class="text-danger">There were errors in your form. Please correct them.</p>
+      {% endif %}
+      {% for error in form.non_field_errors() %}
+        <p class="text-danger">{{ error }}</p>
+      {% endfor %}
+    </div>
+    <form action="." method="POST">
+      {% csrf_token %}
+      <div class="frame__body">
```

Comment:
```suggestion
      <div class="frame__body" style="overflow-y:initial">
```
Prevents the tags selection dropdown from overflowing the div and making it scroll.

Reply:
What happens instead?

Reply:
The drop down list overlays on top of whatever happens to be below it (in this case, the footer). No extra space is created.